src/pivotal-ui/components/forms/forms.scss

```diff
@@ -1475,14 +1475,49 @@ For an organization name input:
 ---
 title: Constraint Validation
 name: form_constraint_validation
-parent: 00_forms
+parent: form_validation
 ---
 
 Modern browsers understand when a form is incorrectly submitted. The constraint
 validation API allows use to take advantage of this to style our own forms in
-the way we see fit.
+the way we see fit. The contraint validation API is built-in to modern browsers
+and is incredibly comprehensive.
+
+An example of when you might want to do this, a required input can have the required
+attribute added to it:
+
+<code><input type="text" name="needed-input" required></code>
```

Comment:
Oh, you mean the ```html_example ``` piece,

I didn't do that because I didn't think it was necessary - but I gladly will.